### DISCUSSION TITLE
Spam

### DISCUSSION THREAD

**Discussion by dermitdemdino** [tutao/tutanota#8180] [Q&A] [answered]
I really like Tuta and have been using it for a long time. Almost everything is perfect. What is annoying, however, is that a lot of spam comes through on some days (addresses of your own domain). I used to use the domain with OVH and also mailbox.org, but that wasn't the case. Does it actually help if I use “Mark as spam”? Because similar emails come through again on other days.

**Comment by charlag** [comment, marked as answer]
Hi, we are sorry to hear you have issues with spam filtering. We already plan some improvements.
In the short term you might find Spam Rules useful

**Comment by dermitdemdino** [comment]
Thank you for your answer. As always, I trust that you will handle this with your usual care and I am sure that a solution will be found.

**Comment by dev-inside** [comment]
+1 Same experience. Tuta works well, but the Spam-Filter works not at all. Sorry to tell, but my webhoster did there even a better job.

**Comment by joshcorne** [comment]
I am facing the same issues. I have a publicly available email address which spammers are more than happy to scrape and send their spam. I have migrated from Google Workspace and hadn't even realised that this many people were sending to my public address because it was correctly caught, until I moved to Tuta. Would be great if this could be resolved as the spam filter is catching legitimate emails and missing blatant spam. This really needs addressing.

**Reply by charlag** [reply]
Hi, thank you for your feedback, spam improvements are needed and planned!
